common/lib/xmodule/xmodule/modulestore/tests/test_xml.py

```diff
@@ -133,7 +133,7 @@ def test_dag_course(self, mock_logging):
         self.assertIsNotNone(parent, "get_parent failed to return a value")
         parent_loc = course_key.make_usage_key('vertical', 'vertical_test')
         self.assertEqual(parent.location, parent_loc)
-        self.assertIn(shared_item, parent.get_children())
+        self.assertIn(shared_item.location, [x.location for x in parent.get_children()])
```

Comment:
Had to make two changes to the tests which depended on the existence of __eq__. The other is this.